File: python/guessing-game.py
from random import randint

print 'Guess the number!'

secret = randint(1, 100)
guesses = 0

print 'The secret number is: ' + str(secret)

while True:
	guesses += 1

	try:
		guess = int(raw_input('Please input your guess: '))
	except ValueError:
		continue

	if guess > secret:
		print 'Too high'
	elif guess < secret:
		print 'Too low'
	elif guess == secret:
		print 'You win! Guesses: ' + str(guesses)
		break

File: ruby/guessing-game.rb
puts 'Guess the number!'

guesses = 0
prng = Random.new
secret = prng.rand(1..100)


puts "The secret number is: #{secret}."

while TRUE
  guesses += 1

  guess = gets.to_i

  if guess > secret
  	puts "Too high"
  elsif guess < secret
  	puts "Too low"
  elsif guess == secret
  	puts "You win! Guesses: #{guesses}"
  	break
  end
end
